### PR TITLE
Add appveyor.yml

### PR DESCRIPTION
**Pull request by aptxkid**
As we've fixed all the tests of CR on Windows, adding appveyor to CI helps us prevent from regression.

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @aptxkid has signed the CLA. Thanks for the pull request!

File: test/functional/test_cluster_basic.py [hunk 1]
```diff
@@ -28,8 +28,8 @@ def test_basic_directory_configs_end_to_end(self, test_job_config):
             'project_directory': project_dir.name,
         })
         build_id = build_resp['build_id']
-        master.block_until_build_finished(build_id, timeout=10)
-        slave.block_until_idle(timeout=5)  # ensure slave teardown has finished before making assertions
+        master.block_until_build_finished(build_id, timeout=30)
+        slave.block_until_idle(timeout=20)  # ensure slave teardown has finished before making assertions
```
Comment (aptxkid) [inline test/functional/test_cluster_basic.py]: I increased the timeout as on Appveyor we are hitting the timeout often.

**Comment by tjlee0909** [comment]
I'm not familiar with the appveyor config, but lgtm :+1:

**Comment by aptxkid** [comment]
Merging this PR marks the official support for Windows of ClusterRunner!

**Pull request merged**